extensions/2.0/Khronos/KHR_lights_environment/README.md

```diff
@@ -0,0 +1,196 @@
+
+# KHR_lights_environment
+
+## Contributors
+
+* Norbert Nopper, <mailto:nopper@ux3d.io>
+* Rickard Sahlin, <mailto:rickard.sahlin@ikea.com>
+* Gary Hsu, Microsoft, <mailto:garyhsu@microsoft.com>
+* Mike Bond, Adobe, <mailto:mbond@adobe.com>
+* Ben Houston, ThreeKit, <mailto:bhouston@threekit.com>
+
+## Status
+
+Draft
+
+Open issues:
+Should Spherical Harmonics be supported or should clients be forced to evaluate the specular cubemap for diffuse contribution?  
+The question comes down to if resolution supplied by 9 coefficients SH is enough for most usecases?  
+It could also be the case that a scene only wants diffuse contribution.  
```

Comment:
Also the inverse may be true, the scene may only want specular contribution, when it already has lighting data, be it through another lighting extension like `KHR_lights_punctual`, or something external in the runtime.

Reply:
Good point - please see my comment below.